specification/datasets/cost_and_usage/dataset.md

```diff
@@ -0,0 +1,96 @@
+| [Pricing Unit](#pricingunit)                                                  | Dimension   | Mandatory     | True         | String    |
+| [Provider](#providername)                                                     | Dimension   | Mandatory     | False        | String    |
+| [Publisher](#publishername)                                                   | Dimension   | Mandatory     | False        | String    |
+| [Region ID](#regionid)                                                        | Dimension   | Conditional   | True         | String    |
+| [Region Name](#regionname)                                                    | Dimension   | Conditional   | True         | String    |
+| [Resource ID](#resourceid)                                                    | Dimension   | Conditional   | True         | String    |
+| [Resource Name](#resourcename)                                                | Dimension   | Conditional   | True         | String    |
+| [Resource Type](#resourcetype)                                                | Dimension   | Conditional   | True         | String    |
+| [Service Category](#servicecategory)                                          | Dimension   | Mandatory     | False        | String    |
+| [Service Name](#servicename)                                                  | Dimension   | Mandatory     | False        | String    |
+| [Service Subcategory](#servicesubcategory)                                    | Dimension   | Recommended   | False        | String    |
+| [SKU ID](#skuid)                                                              | Dimension   | Conditional   | True         | String    |
+| [SKU Meter](#skumeter)                                                        | Dimension   | Conditional   | True         | String    |
+| [SKU Price Details](#skupricedetails)                                         | Dimension   | Conditional   | True         | JSON      |
+| [SKU Price ID](#skupriceid)                                                   | Dimension   | Conditional   | True         | String    |
+| [Sub Account ID](#subaccountid)                                               | Dimension   | Conditional   | True         | String    |
+| [Sub Account Name](#subaccountname)                                           | Dimension   | Conditional   | True         | String    |
+| [Sub Account Type](#subaccounttype)                                           | Dimension   | Conditional   | True         | String    |
+| [Tags](#tags)                                                                 | Dimension   | Conditional   | True         | JSON      |
+
+<div class='h4-nonindex'>Relationships</div>
+
+(none)
+
+<div class='h4-nonindex'>Requirements</div>
+
+The CostAndUsage dataset adheres to the following requirements:
+
+* CostAndUsage MUST be present.
```

Comment:
This doesn't seem to be needed, right? I'm not sure what it means. If they have the dataset, these rules apply. If they don't have the dataset, then these rules don't apply because they're not even looking at cost and usage data. I'm not sure what this is really conveying as a requirement.

Unless it's saying that the CostAndUsage dataset is required for them to be able to claim FOCUS support? But if that's it, what are they "certifying" is FOCUS-compatible? A dataset or a tool? I believe we can only certify datasets, so I'm not sure this line is needed.